### DISCUSSION TITLE
Proper SVG logo

### DISCUSSION THREAD

**Discussion by loganmarchione** [authelia/authelia#4566] [General]
### Description

I'm reading [this page](https://www.authelia.com/reference/guides/branding/) and downloaded the [SVG logo](https://www.authelia.com/images/branding/logo.svg).

However, upon inspection, the SVG file is actually just an encoded PNG file. Can you please provided a proper SVG file?

```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="640" height="640" viewBox="0 0 640 640">
  <image id="Logo" x="154" y="157" width="332" height="326" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAUwAAAFGCAYAAAAB9aY5AAAgAElEQVR4nOydd3xU1daG37VnJoUugvSmKF4RBEIIISBRkCrl6g32hgoWpNjL53W899o7Kk0FC6ISK6ChqZEWAgQRRCxIUcACKEpLMnP2+n5nkkAy6cneZ85kzvMHJJmZtdc+c+ad3dZ
...
...
...
/lKkdYxl86ngAAAAASUVORK5CYII="/>
</svg>
```

### Use Case

I was trying to find an Authelia SVG logo to use in my network diagram at home.

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)
- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
Partially done in fb62c33129d15fc337bd8e086a1ab67d9d079caf

I'll try and get the color accuracy higher.

**Reply by james-d-elliott** [reply]
Made significant improvements in aa61058ca9406e9c628fd0a6dd9ddaa82cf0f715

There are some vectors which are slightly off  but they are mostly not noticeable.

**Reply by loganmarchione** [reply]
Thank you!
